README.md

```diff
@@ -274,7 +276,6 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Git Immersion](https://gitimmersion.com/) 
 - [30SecondsOfCode Git Snippets](https://www.30secondsofcode.org/git/p/1)
 - [Git - the simple guide](https://rogerdudler.github.io/git-guide/)
```

Comment:
Thank you for making all the changes @ashusengar but please add a new line here (without the dash, just an empty new line).